.github/workflows/release.yml

```diff
@@ -100,21 +100,33 @@ jobs:
             .pages/firefox/updates.json \
             > .pages/firefox/updates.json.tmp && mv .pages/firefox/updates.json.tmp .pages/firefox/updates.json
 
+      - name: create update manifest for Thunderbird
+        run: |
+          VERSION=$(echo ${{ github.ref_name }} | cut -c 2-)
+          DIGEST="sha256:$(sha256sum build/linux_entra_sso-${VERSION}.thunderbird.xpi | cut -d ' ' -f 1)"
+          LINK="https://github.com/siemens/linux-entra-sso/releases/download/v${VERSION}/linux_entra_sso-${VERSION}.thunderbird.xpi"
+          jq --arg version "${VERSION}" --arg digest "${DIGEST}" --arg link "${LINK}" \
+            '."addons"."@linux-entra-sso.tb"."updates" += [{"version":$version, "update_link":$link, "update_hash":$digest}]' \
+            .pages/thunderbird/updates.json \
+            > .pages/thunderbird/updates.json.tmp && mv .pages/thunderbird/updates.json.tmp .pages/thunderbird/updates.json
```

Comment:
That's for the time are still no official add-on, right?

Reply:
Right. We probably could get it into the store as the "limited audience" requirement was dropped. However I want to keep things consistent between FF and TB and also the TB store currently lacks a documented API to automate the distribution.

As we don't need a signature for the TB store at the moment, TB simply does not know about the extension and we can later on upload it with the same id.